src/app/components/home/home.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "src/styles/_variables.scss";
+@import "src/styles/_colors.scss";
+@import "src/styles/_mixins.scss";
+
+.home-container {
+    margin: $sp-137 $sp-150 0 $sp-150;
+    padding: 0;
+}
+.home-top {
+    margin-bottom: $sp-14;
+}
+.add-task-label {
+    font-size: $sp-30;
+}
+.create-button button {
+    color:$white;
+    @include buttonCard($sp-8);
+    background-color:$blue;
+}
+.task-cards-container { 
+    display: grid;
+    grid-template-columns: repeat(3, 1fr); /* you can change 1fr to whatever you want, 200px for example*/
+    gap: 2rem;
+}
+#add-one-task-icon {
+    @include displayAlign(flex, center, center);
+    margin-top: $sp-36;
+}
+#add-one-task-icon button{
+   background-color: #F8F9FF;
+   border: none;
+
+}
+.task-card {
+    @include buttonCard($sp-14);
+}
+#task-name {
+    margin-top: 0;
+    color: #32394B;
+}
+#task-time {
+    font-size: 14px;
+    line-height: 16px;
+    color: $secondaryTextGrey;
+}
+.task-card-buttons {
+    display: flex;
+}
+.task-card-buttons button {
+    border: none;
+    margin-right: 26px;
+    width: 24px;
+    background-color: $white;
+   
+}
+#task_name_input {
+    @include buttonCard($sp-5);
+    border: $sp-3 solid $borderGrey;
+}
+#add-task-button {
```

Comment:
lets use classes instead of IDs as selectors.

Reply:
implemented as classes instead of IDs.